omero/users/clients-overview.txt

```diff
@@ -56,7 +56,12 @@ views.
     :alt: OMERO.web image viewer
     
     OMERO.web image viewer
-    
+
+OMERO.web now features almost all of the functionality of OMERO.insight
+including the ability to administrate users and groups. Work is ongoing to
```

Comment:
OMERO.web had the ability to manage users and groups for a long time (long before insight)